features/step_definitions/hospital_listing_steps.rb

```diff
@@ -0,0 +1,5 @@
+Given("the following hospital exists") do |table|
```

Comment:
This should go in a file called something like "new_object_steps.rb"

Reply:
Modified